### ISSUE TITLE
Redis 消息订阅网上都说靠不住，为什么这里还要使用他来作为分布式消息传递

### ISSUE THREAD

**Issue by mytwz** [socketio/socket.io-redis-adapter#399]
Redis 消息订阅网上都说靠不住，为什么这里还要使用他来作为分布式消息传递
这个问题最近把我给坑死了

由于消息丢失导致部分客户端没有收到房间消息
由于消息丢失导致　remoteJoin 没有收到反馈而超时触发报错

**Comment by darrachequesne** [comment]
Sorry, I'm not able to understand the question.

Could you please explain your issue in plain English? Thanks!

**Comment by mytwz** [comment]
Redis news subscription is unreliable on the Internet, why is it used here as a distributed message delivery
This problem pitted me to death recently

Some clients did not receive the room message due to message loss
Due to the loss of the message, 　remoteJoin did not receive feedback and timed out to trigger an error
